src/components/WhimsicalInstaller/WhimsicalInstaller.js

```diff
@@ -67,21 +67,21 @@ class WhimsicalInstaller extends PureComponent<Props, State> {
   }
 
   getHeight = () =>
-    // Our height will be 1/3rd of our width.
-    // This is so we end up with 3 squares:
-    //  _____________________
-    // |      |      |      |
-    // |      |      |      |
-    // |______|______|______|
+    // Our height will be 1/2 of our width.
+    // This is so we end up with 2 squares:
+    //  ______________
+    // |      |      |
+    // |      |      |
+    // |______|______|
```

Comment:
I was building this with the mistaken assumption that it'd be about 600px wide. Once I actually put it in the modal, it needed to be 420ish px, and I realized that the Planet and Folder were too far apart, it looked really funny. So I needed to reconfigure their distance from each other